resources/build/l10n/parse-crowdin.sh

```diff
@@ -0,0 +1,163 @@
+#!/bin/bash
+# Extract Keyman translations from https://crowdin.com/project/keyman
+# into /crowdin/ and move files to corresponding projects.
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
+. "$(dirname "$THIS_SCRIPT")/../build-utils.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+display_usage() {
+  echo "parse-crowdin [-all] [-android] [-common] [-developer] [-ios]"
+  echo "                     [-linux] [-mac] [-web] [-windows]"
+  echo
+  echo "Extract Keyman.zip at root of the repo and then copy files to given platforms"
+  echo "Options:"
+  echo "  -all            Copy localization files for all platforms"
+  echo "  -android        Copy localization files for Android"
+  echo "  -common         Copy localization files for common"
+  echo "  -developer      Copy localization files for developer"
+  echo "  -ios            Copy localization files for ios"
+  echo "  -linux          Copy localization files for linux"
+  echo "  -mac            Copy localization files for mac"
+  echo "  -web            Copy localization files for web"
+  echo "  -windows        Copy localization files for windows"
+  exit 1
+}
+
+# Copy a file to a destination directory. If destination doesn't exist, create it
+# $1 is the file to copy
+# $2 is the destination directory
+function copy_file() {
+  if [ ! -d "$2" ]; then
+    echo "Creating $2"
+    mkdir "$2"
+  fi
+  echo "Copying $1"
+  cp "$1" "$2"  
+}
+
+function process_android() {
```

Comment:
Let's put the platform specific code into its own file for maintenance and `. parse-crowdin-android.sh` and then call `process_android` as normal.